app/about_us/ourValue/OurValues.js

```diff
@@ -0,0 +1,68 @@
+import styles from "./ourValue.module.css";
+
+// TASK - React 1 week 1
+// After you are finished with creating the page, move the OurValues, OurCrew, OurPartners components into their own files
+// OurValues.js, OurCrew.js, OurPartners.js should live in this folder
+// import and use the components from the newly created files
+export default function OurValues() {
+  // TASK - React 1 week 1
+  // Create the "Our Values" section
+  // Use the descriptions provided in /app/about_us/README.md
+  // Some inspiration ideas found in /data/inspiration_about_us
```

Comment:
These comments can be deleted in the upcoming HWs :)

Reply:
in the week 3 i'll delete all comments